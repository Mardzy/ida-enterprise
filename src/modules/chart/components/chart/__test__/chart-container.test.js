import ChartWrapper from "../chart";

describe("The chart wrapper", () => {

  // TODO: add more tests

  it("Renders an empty shell", () => {
    const wrapper = mount(<ChartWrapper/>);

    expect(toJson(wrapper)).toMatchSnapshot();
  });

});