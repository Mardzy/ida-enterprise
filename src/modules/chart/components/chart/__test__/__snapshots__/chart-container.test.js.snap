// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The chart wrapper Renders an empty shell 1`] = `
.c0 {
  margin-bottom: 2rem;
}

.c0:last-child {
  margin-bottom: 0;
}

<Chart
  config={Immutable.Map {}}
  id={null}
  renderGraph={[Function]}
  title=""
>
  <Styled(Component)
    className="chart"
    whitespaceAfter={2}
  >
    <StyledComponent
      className="chart"
      forwardedComponent={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "attrs": Array [],
          "componentStyle": ComponentStyle {
            "componentId": "sc-bdVaJa",
            "isStatic": false,
            "lastClassName": "c0",
            "rules": Array [
              "
  margin-bottom: ",
              [Function],
              ";
  
  &:last-child {
    margin-bottom: 0;
  }
",
            ],
          },
          "displayName": "Styled(Component)",
          "foldedComponentIds": Array [],
          "propTypes": Object {
            "whitespaceAfter": [Function],
          },
          "render": [Function],
          "styledComponentId": "sc-bdVaJa",
          "target": [Function],
          "toString": [Function],
          "warnTooManyClasses": [Function],
          "withComponent": [Function],
        }
      }
      forwardedRef={null}
      whitespaceAfter={2}
    >
      <Component
        className="chart c0"
        whitespaceAfter={2}
      >
        <section
          className="chart c0"
        >
          <Header
            as="h3"
            attached="top"
          >
            <h3
              className="ui top attached header"
            />
          </Header>
        </section>
      </Component>
    </StyledComponent>
  </Styled(Component)>
</Chart>
`;
